src/main/java/com/ericsson/ei/handlers/EventToObjectMapHandler.java

```diff
@@ -83,28 +83,29 @@ public void setJmesPathInterface(JmesPathInterface jmesPathInterface) {
 
     public void updateEventToObjectMapInMemoryDB(RulesObject rulesObject, String event, String objectId) {
```

Comment:
I would like javadoc added to this method also.

Reply:
Will add.